index.html

```diff
@@ -8,9 +8,9 @@
   </head>
   <body>
     <div id = "title-side">
-      <h1><span class= "title">Linked</span><img src = "images/linked-list-logo (1).svg" id = "logo" alt= "logo"/><span class = "title">List</span></h1>
-      <input type = "text" id = "input-title"></input>
-      <input type = "text" id = "input-url"></input>
+      <h1><span class= "title">Linked</span><img src = "images/linked-list-logo (1).svg" id = "logo" alt= "logo" height = "50px" width ="50px"/><span class = "title">List</span></h1>
```

Comment:
My inclination would be to put the sizing in our CSS rather than in the HTML - I'm fine with it if you prefer it this way!